scene/3d/physics_body_3d.cpp

```diff
@@ -1270,7 +1270,7 @@ void CharacterBody3D::_move_and_slide_grounded(double p_delta, bool p_was_on_flo
 
 	for (int iteration = 0; iteration < max_slides; ++iteration) {
 		PhysicsServer3D::MotionParameters parameters(get_global_transform(), motion, margin);
-		parameters.max_collisions = 4;
+		parameters.max_collisions = 6;
```

Comment:
There should be a comment to explain why the value of 6 was chosen. You can use the same description as this PR.